src/icons/tokens/ct_xtk8rSz9suPb6D6VLquyfVji25FcnFRDjn3dnn5mmvHsPiESt.svg

```diff
@@ -0,0 +1,22 @@
+  <path fill="#fff" d="M573.16 633.01c1.45 0 2.39 1.12 3.35 3.4 2.2 5.2 1.9 7.8-1.2 11.1-7.17 7.56-61.11 36.53-67.78 36.53-.12 0-.23-.01-.32-.03-1.8-.4-10.2-7.3-10.2-8.4 0-.4 3-2.1 6.8-3.9 28.5-13.4 47.7-24.2 63.1-35.5 2.91-2.12 4.83-3.2 6.25-3.2m0-8c-3.72 0-6.95 1.81-10.96 4.74-14.89 10.93-33.94 21.63-61.79 34.73-6.38 3.02-11.4 5.4-11.4 11.14 0 3.91 2.37 6.55 7.34 10.64 5.9 4.85 8 5.32 9.12 5.57.67.15 1.36.22 2.05.22 1.74 0 7.05 0 37.5-16.38 7-3.77 30.31-16.54 36.09-22.64 6.75-7.19 5.18-14 2.76-19.72-1.59-3.79-4.75-8.3-10.71-8.3z"/>
+  <path d="M368.1 692.46c-11.77 0-22.88-.82-31.3-2.31-4.85-.77-9.76-1.34-11.6-1.34-3.34 0-12.2 0-12.2-9.6v-3.5l6.4-3.2 2.8 2.1c3.29 2.42 11.11 3.25 36.39 3.8 5.17.14 10.13.21 14.74.21 26.66 0 33.91-2.21 35.5-2.89l3.45-1.42 5.48 4.25-.5 3.6c-.2 1.36-.76 4.55-1.63 5.87l-.5.61c-1.14 1.14-2.42 1.56-7.77 1.77-2.28.09-10.53.62-20.08 1.29-6.09.5-12.56.76-19.18.76.01 0 .01 0 0 0z"/>
+  <path fill="#fff" d="m317 675.71 2.8 2.1c4.2 3.1 11.4 4 38.7 4.6 5.24.14 10.2.21 14.83.21 18.79 0 32.18-1.12 37.07-3.21l3.4-1.4-.5 3.6c-.3 2-.8 3.9-1 4.2-.3.3-2.6.5-5.1.6-2.6.1-11.7.7-20.2 1.3-6.12.5-12.55.75-18.9.75-11.29 0-22.28-.78-30.6-2.25-4.4-.7-9.9-1.4-12.3-1.4-6.2 0-8.2-1.4-8.2-5.6v-3.5m4.8-6.4-12.8 6.4v3.5c0 6.09 2.85 9.33 5.25 10.97 3.38 2.32 7.43 2.63 10.95 2.63 1.4 0 5.89.48 10.98 1.29 8.63 1.52 19.97 2.36 31.93 2.36 6.71 0 13.27-.26 19.5-.77 9.49-.67 17.67-1.2 19.9-1.28 4.87-.19 8-.49 10.45-2.94l1-1.22c.36-.54 1.47-2.2 2.25-7.45l.01-.09.5-3.6-10.97-8.5-3.4 1.4-.09.04c-.62.26-6.8 2.57-33.94 2.57-4.58 0-9.5-.07-14.62-.2-19.58-.43-31.44-1.06-34.14-3.03l-2.76-2.08z"/>
+  <path d="M363.36 716.38c-1.91 0-3.59-.03-5-.08-10.75-.39-28.91-2.08-32.75-3.28-4.43-1.41-9.61-11.36-9.61-15.62 0-1.8 0-6.6 4.3-6.6.29 0 .29 0 3.33.38 2.25.28 5.62.7 9.35 1.15 6.61.83 29.49 1.51 51.03 1.51 8.84 0 16.39-.12 21.83-.34.5-.03 1.01-.05 1.45-.05 5.78 0 6.27 4.49 6.49 6.41.6 6.73-4.09 13.09-10.44 14.27-6.26 1.17-27.83 2.25-39.98 2.25z"/>
+  <path fill="#fff" d="M320.3 694.81c.1 0 5.6.7 12.2 1.5 7.29.92 31.16 1.54 51.5 1.54 8.4 0 16.21-.11 22-.34.49-.03.92-.05 1.29-.05 2.01 0 2.26.57 2.51 2.85.4 4.5-2.9 9.1-7.2 9.9-5.68 1.05-26.64 2.17-39.24 2.17-1.82 0-3.46-.02-4.86-.07-11.1-.4-28.5-2.1-31.7-3.1-2.2-.7-6.8-8.6-6.8-11.8 0-1.4.1-2.6.3-2.6m0-8c-3.68 0-6.83 2.43-7.84 6.06-.16.57-.27 1.18-.34 1.88-.08.75-.12 1.65-.12 2.66 0 3.75 1.99 7.97 3.65 10.84 3.69 6.38 6.75 7.95 8.72 8.58 4.66 1.46 23.73 3.11 33.84 3.47 1.46.05 3.19.08 5.15.08 12.43 0 34.08-1.09 40.69-2.31 8.34-1.55 14.48-9.84 13.72-18.48l-.02-.18c-.17-1.54-.46-4.11-2.33-6.43-1.9-2.35-4.63-3.53-8.13-3.53-.49 0-1.01.02-1.69.06-5.39.22-12.85.33-21.59.33-21.4 0-44.05-.66-50.5-1.48-3.75-.45-7.11-.87-9.36-1.15-3.31-.4-3.31-.4-3.85-.4z"/>
+</svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" viewBox="0 0 736.8 720.4"><path d="M225.7 684.2c-3.8 0-8.3-1.5-16.6-5.4-17.4-8.2-48.5-27-55.6-33.3-5.3-5-3.5-11.9-.9-19a405 405 0 0 0 4.9-14.6l1.7-5.5-4-3.1a371 371 0 0 1-34.7-34.3c-2.5-3-3.5-3.3-4.9-3.3-1.3 0-3.2.4-6.2 1.3-16.6 5.1-22 6.7-25.8 6.7-3.8 0-5.7-1.7-7.3-3.5-8.7-9.4-40.3-61.5-40.3-70 0-2.4 1.5-5.4 8.6-12 8.3-7.8 14.8-14.5 17-17.2a114.1 114.1 0 0 0-1.5-4.4c-1.7-5.1-6-18.7-13-41.8-1-3.1-2.3-8-2.9-10.7-1.2-.5-4-1.4-11-2.8-5.4-1-23-4.7-25.7-9.4v-.1c-1.7-3-3.5-11.5-3.5-49.5v-4.6C4 319 4 316 17.1 313l1.6-.4 14.2-3.5 4-1 .8-6.4c2.4-17.5 6.9-38.1 11.9-53.7l2.7-8.2-.4-.3c-2.9-2.1-7-6-10-8.8-6.9-7-6.9-8.2-6.9-9.6 0-4 14.7-32 17.4-36.9a334.2 334.2 0 0 0 4.7-8c.4-1.2 2.2-4 9.6-15.1 7.2-11 10.1-15.4 15.6-15.4 2 0 3.7.6 5.8 1.4a88.4 88.4 0 0 0 13.1 3.5c.9-.6 2.7-2.3 4.8-5.1l.1-.1a434.3 434.3 0 0 1 47.6-46.9 153 153 0 0 0-2.1-7.8v-.2c-4.4-15.5-3.7-17.3 11-27.1l.2-.2a205.5 205.5 0 0 1 25.2-15c.3-.2 29.4-16.5 33.8-16.5 4.5 0 8.2 3.7 8.2 8 3.8.3 6.3 2.2 10.2 6.1l3.2 3.1 8.6-3c12.8-4.6 35-10.2 52.8-13.3l10.7-2 .3-4.6c.2-2 .7-6.8 2.8-9 .4-.3 1-.7 1.9-1.1a5.6 5.6 0 0 1-1.3-4c.2-2.8 2.3-5.5 5.4-6.8 1-.5 2.7-1.2 19-1.2 18.8 0 45.6 1 53.6 2.4 10.7 2 12 2.7 12.8 3.4.8.7 1.9 2 3.9 10.1 1.8 6.5 2.6 9.7-.7 12.5 9.2 1.6 18.3 3.6 29.9 6.4a336 336 0 0 1 40 12.8l1.9.8 6.5 2.7c25.2 10.2 65 34.3 80.6 48.7 30.7 28.4 50.3 51.1 65.5 76 21.6 35 35.3 70.9 41.9 109.5 1.2 7.4 3 17.5 4 21.6 2.4 11.1 2.5 46.5.2 66.5a321.5 321.5 0 0 1-83.1 180h.1a5 5 0 0 1 3.9 1.7c1.6 2 1.5 4.6.8 8.2-1.1 6.5-4.3 11-13 18.7a408.6 408.6 0 0 0-16.6 15.5c-2.7 2.7-6.1 6-9.3 8.8h.6c1.4 0 2.4.4 3.1.7 2.3 1 4 3.2 5 6.2 1 3.5.6 7.4-1.2 10.1-3 4.6-20.4 15.4-28.8 19-2.7 1-6.1 2.5-9.4 3.9a656.6 656.6 0 0 1-17.7 7c-.7.2-3.2 1.4-5.3 3a50 50 0 0 1-9.5 5c-2 .7-4.6 2.3-5.8 3.2-3.7 3.4-5.2 3.4-6 3.4-1.5 0-2.5 0-9.5-7.2-2-2.2-4-3.9-5-4.6l-8.4 2.5a358.7 358.7 0 0 1-41.8 11.1c-7.6 1.4-7.8 1.4-8.9 3.6a4.1 4.1 0 0 1-3.7 2.4c-.4 0-1.6 0-4-1.4-.2 0-.9-.2-2.7-.2-2.3 0-5.6.2-11.3.6a530.9 530.9 0 0 1-70.6-.8 343.7 343.7 0 0 1-64-14.9 34 34 0 0 0-8-1.7c-1.6 0-2.7.3-5.8 3.5l-3.8 3.9-4.7.8-3-1.5c1 .6 1.8 1.5 2.3 2.8 1 3.3-1 6.2-3.5 9-1.7 2-4.6 4.5-9.4 4.5zm7.2-17.6c.5 0 1.1.2 1.8.5l-.2-.1a54.7 54.7 0 0 0-6-2.8c-.7-.2-3.2-.9-4-3.4-3.7-.1-6.9-1.1-10.3-3.2a61.2 61.2 0 0 0-8.8-4l-.9-.3a442.5 442.5 0 0 0 28.4 13.3zm44.8-522a300.6 300.6 0 0 0-64 30.2 233.9 233.9 0 0 0-95 130.2c-6 21.8-7.8 34.8-8.4 58-.7 29 1.7 49.6 8.7 73.5a259.8 259.8 0 0 0 47 89.2 242 242 0 0 0 80.7 62.2c39.3 18.7 77 27.1 122.5 27.3 7.5 0 13.3.2 17.2.3 2 .1 3.9.2 5.4.5 1.7-1 4.3-1.3 8.7-1.7 36.7-3 43.5-4.3 60-10.7a2126 2126 0 0 0 20-7.8 356.8 356.8 0 0 0 81.7-45.8 321.3 321.3 0 0 0 45-43A248 248 0 0 0 661 375a113.2 113.2 0 0 1 3.7-23.4c.9-4.6 2.7-9.1 4.6-11.4 3.3-3.9 4-29.7-2.2-59.9a409.2 409.2 0 0 0-15.5-50 311.7 311.7 0 0 0-44-74.2c-9.2-11.7-36.9-39.4-41-41.6-2-1-5.4-3.4-8.1-5.7-9.4-7.6-26.8-18.1-43-26.1l-3.8-1.8a5.2 5.2 0 0 1-3 3l2.2 1.6.2.1c3.6 2.6 5 4.2 5 6.3 0 2.8-2.2 4.9-5.2 4.9-1.4 0-3.9-.4-9.7-3.8a335.3 335.3 0 0 0-51.7-22 252.4 252.4 0 0 0-66-13.8c-7.1-.8-17.6-1.2-29.4-1.2-16.5 0-32.4.8-41.5 2.2a190.4 190.4 0 0 1-19.8 2.5c-5.5 0-5.8-4-5.8-4.8 0-2.5 2.2-5 7-5 .9 0 3.6-.6 5.7-1.4 2.4-.8 6.1-1.7 8.3-1.7.6 0 2-.5 2.5-.9.6-.6 1.4-2.7 1.7-4.4l-2 .4-16 3c-15.2 3-31.4 7.3-43.2 11.7l1 1c4.1 4.2 8 8.1 4.8 11.4-.8.8-1.1 1.1-8.2 3a71 71 0 0 0-3.8 1.3c1.2.3 2.3 1 3 2.2.7 1.1.8 2.4.4 3.5-.3 1-.8 2.8-6.8 5.5a291.1 291.1 0 0 0-46.5 25l-10.3 6.2c-11 6.7-17.4 11.3-24.8 17.5a30 30 0 0 1-7.5 5c1 6.3-9.5 17-10.5 18-2 1.9-3.5 3.4-5.9 3.4-.7 0-1.4-.1-2-.4-1 2-2.7 4.8-5.2 9-4.2 6.8-9.4 14.9-14 21.6-4.5 6.6-9.7 15-11.5 18.7-2 3.7-4.8 8.8-6.5 11.2-6 9-16.2 32.1-21.7 49.3-2.6 8.2-4.2 16.8-4.7 21.3l.6-.7c6-6.8 8.4-11.7 13.3-27.5 2.6-8.1 5.9-17.7 7.2-20.5l4.4-11.2c12.1-31.9 37-67 65-91.6 37.8-33.3 83.5-56 132.2-66 15-3 35.1-4.6 56.5-4.6 21.8 0 42 1.7 56.9 4.9 23.2 4.9 61.9 21.7 67 31h.2c1.5 3 2.5 5.7-.9 9-4.5 4.6-17.1 4.9-20.9 4.9l-6.9-.3a253.6 253.6 0 0 0-75.6 5.5c-16.2 4.6-16.9 4.8-33.6 12.5l-12.2 5.2-7.3 3a276.6 276.6 0 0 0-46.5 22.7c-2.6 1.5-14.8 10-22 15.5a343 343 0 0 0-39.3 38.1c-6.9 7.6-8.2 7.6-9.6 7.6-2 0-3.5-1-4.3-2.6-1.4-2.8-.6-6 7.9-16.9a225 225 0 0 1 49-45.3c8.7-6 20.5-14.4 27.3-19.8l-1 .3zM61.1 247.1c-2.8 7.9-7.6 26.2-10.4 42.2 2.2-9.8 5.7-21.4 9.6-32.7l2.4-7.4-1.6-2.1zm51.7-92.7 4.3 1.2c.9-.5 3-2 6.6-5.6 5.8-6 9.3-8.1 11.7-8.8 1.3-4.3 5.8-9.1 10-12a89.2 89.2 0 0 0 16.2-15.7 20 20 0 0 1-3.3-4.3c-3.4 2.7-9.8 8.3-18.9 16.6a395.8 395.8 0 0 0-26.6 28.6zM419.6 47l1 .4c10.2 2 39 9.5 47.4 12.3 4.3 1.5 7.4 2.4 9 2.4.5 0 .7 0 1.3-1l-4.6-1.5a316 316 0 0 0-54.8-15l.7 2.4z"/><path fill="#fff" d="M343.6 8c18.2 0 45.1 1 53 2.3a64 64 0 0 1 10.7 2.5c.5.4 1.7 4.1 2.7 8.1 2 7 2 7.5.2 8.7-.7.6-3 1-6.8 1-5.3 0-13.5-.7-24.4-1.9-4.6-.5-10-.7-14.7-.7-6.6 0-11.8.4-11.6 1.3.2.6 7.6 1.6 16.5 2.2a371 371 0 0 1 112.3 23.8l8.5 3.5c24.5 10 64.3 34 79.4 48 30.4 28.1 49.8 50.6 64.8 75A297.1 297.1 0 0 1 675.5 290a396 396 0 0 0 4.1 21.8c2.3 10.6 2.4 45.4.1 65.1a318.6 318.6 0 0 1-56.6 146.4c-17.3 24.1-47 56-63.6 68.1-13 9.5-13.9 10.3-12 11h.2c3.6 0 23.3-15.1 40.6-31.5 7-6.5 10.7-9.6 12.4-9.6 1.4 0 1.5 1.8.8 5.3-1 5.5-3.8 9.4-11.8 16.4-4 3.5-11.5 10.5-16.7 15.7a308.2 308.2 0 0 1-15.5 14.3c-6.6 5.5-16.7 16-16 16.6h.2c1.5 0 11.5-6 18.1-11 4.1-3.2 5.9-4.3 7.3-4.3.5 0 1 .2 1.5.4 3 1.3 4.2 7.2 2.1 10.4a103.9 103.9 0 0 1-27 17.5c-4.6 1.8-11.8 5-16.2 6.7-4.4 2-9 3.7-10.5 4.1s-4.3 2-6.5 3.5-6 3.6-8.5 4.5-5.6 2.8-7 4c-1.5 1.3-3 2.4-3.5 2.4s-3.4-2.7-6.6-6c-3-3.3-6.4-6-7.2-6s-5.4 1.3-10.3 3a351.5 351.5 0 0 1-41.3 11c-9 1.5-9.9 2-11.8 5.8a6 6 0 0 1-2-.9c-.9-.5-2.2-.7-4.8-.7-2.5 0-6.2.2-11.6.6a538.5 538.5 0 0 1-69.9-.8 340 340 0 0 1-63.2-14.7c-4.2-1.3-7-2-9.3-2-3.3 0-5.4 1.5-8.6 4.8l-3.8 4-4.8-2.5a53.6 53.6 0 0 0-6.7-3c-1.5-.5-1.6-.9-.6-2 1.1-1.4.6-1.6-3.5-1.6-3.4 0-6.2-.8-9.2-2.6-2.4-1.4-6.7-3.4-9.6-4.4l-14.7-5.4a47 47 0 0 0-12-3.4c-5 0 7.8 8.7 31.8 20.4a220 220 0 0 0 20.2 9.1c3.8.7 3.8 2.4.1 6.5-1.9 2.1-3.8 3.2-6.4 3.2a40 40 0 0 1-14.9-5 364.5 364.5 0 0 1-54.6-32.7c-2.9-2.7-2.8-6.4.2-14.7a421 421 0 0 0 5-14.7l2.4-8.2-6.2-4.8a371.3 371.3 0 0 1-34-33.6c-2.7-3.3-4.7-4.8-8-4.8-1.9 0-4.2.5-7.4 1.5-15 4.6-21.3 6.5-24.6 6.5-2.2 0-3-.8-4.3-2.2-9.8-10.6-39.3-61-39.3-67.3 0-1.5 2.4-4.5 7.3-9 9.6-9 18.7-18.5 18.7-19.4A1808 1808 0 0 1 51 423.6c-1.6-5.4-2.9-10.4-2.9-11.2 0-1.7-3.7-3-14-5-11.6-2.2-21.9-5.6-23-7.5-2-3.8-3-19.9-3-47.6 0-34.7-.6-32.9 11.6-35.7L34 313l6.4-1.7 1.3-9c2.3-17.4 6.8-37.7 11.7-53l3.6-11-2.8-2a81.8 81.8 0 0 1-15.2-15 301.3 301.3 0 0 1 31-57.9c7-10.5 9.2-13.7 12.3-13.7 1.2 0 2.6.5 4.4 1.2a77.2 77.2 0 0 0 15.2 4c1.1 0 4.2-2.9 7.3-6.9 7.7-9.4 29.6-31.4 40.1-40 4.8-4 8.7-7.8 8.7-8.4 0-.5-1-5-2.6-10-3.9-14-3.9-13.9 9.6-23a208.6 208.6 0 0 1 25-14.9 248 248 0 0 1 31.8-15.9c2.2 0 4.2 2 4.2 4 0 1.1-2.3 2.7-6.7 4.5-13.2 5.6-53.3 29-53.3 31.2 0 .3.2.5.7.5 2 0 8.8-3.5 19.3-9.9 21-12.9 39.2-22.3 43.2-22.3 2.7 0 4.3 1 8.2 5l5 4.8 11-3.9a364.1 364.1 0 0 1 52.1-13c6.6-1.2 12.8-2.5 13.7-3s5.6-.9 10.4-.9 8.5-.3 8.1-.6c-.3-.3-4.6-.7-9.4-.8h-8.8l.3-5c.2-2.9 1-5.7 1.6-6.4 1.8-1.8 16.1-3.9 32.1-4.6 17-.8 27-1.6 27.4-2.2.3-.2 0-.9-.6-1.5s-6.1-.8-13.3-.8c-13.7 0-33.3.9-37 2.1-2 .6-3.8 1-5 1-.5 0-.8 0-1-.2-1.8-1-.5-3.8 2.2-5 1.3-.6 8.3-.8 17.4-.8M227 80.8c1.3 0 3.4-.5 6.1-1.5 2.8-1 6-1.5 8.3-1.5 1.6 0 2.8.3 3.1.8.3.5-1.8 2-4.7 3.2a294.7 294.7 0 0 0-35.7 18.3l-21.5 13.2c-11.4 7-18 11.7-25.3 17.8-2.3 2-5.2 4-6.5 4.3-2.9 1-3.9 2.5-2.5 4.1 1.2 1.5-3.9 9.7-9.3 14.8-1.6 1.6-2.4 2.3-3 2.3-.4 0-.8-.4-1.3-1-.9-1.2-1.6-1.7-2-1.7a296.6 296.6 0 0 1-21.2 34.6c-4.6 6.7-9.9 15.3-11.8 19.2-2 3.8-4.7 8.6-6.2 10.7a250.3 250.3 0 0 0-22.2 50.4c-3.3 10.4-5.8 24-4.9 26.4.4 1 1.1 1.6 2 1.6 1.6 0 3.6-1.3 5.6-3.6 6.5-7.3 9.1-12.7 14.1-28.9 2.8-8.8 6-17.7 7-20 .5-1 2.5-6.3 4.5-11.5a245.4 245.4 0 0 1 64-90 281.6 281.6 0 0 1 130.3-65c15.3-3 35.5-4.6 55.7-4.6 20.5 0 40.9 1.6 56 4.8 23.3 5 60.3 21.6 64.5 29 1.3 2.5 1.3 3 0 4.3-2.3 2.2-9.8 3.7-18.2 3.7-2.1 0-4.3 0-6.5-.3a256.6 256.6 0 0 0-77 5.6 156.7 156.7 0 0 0-34.3 12.8c-3 1.4-11.8 5-19.4 8.2a278.6 278.6 0 0 0-46.2 22.5 263 263 0 0 0-23 16.2c-10 7.6-20.7 18-39.9 38.5a35.3 35.3 0 0 1-6.7 6.3c-2.3 0 0-4.4 6.7-13a219.6 219.6 0 0 1 48.1-44.5c28-19.5 34.4-24.7 34.4-27.6 0-1.1-.2-1.7-1.3-1.7-1.5 0-4.3.8-10.2 2.7a305.4 305.4 0 0 0-65 30.7A237.5 237.5 0 0 0 115 304c-6 22-8 35.4-8.6 59-.7 29.2 1.7 50.1 8.8 74.7a263 263 0 0 0 48 90.7 245.7 245.7 0 0 0 82 63.2 279 279 0 0 0 124.1 27.7c21.8.1 28.3.8 18.1 2a235 235 0 0 1-22 .3h-5.4c-12.7 0-16.3.7-8.7 1.8a187.9 187.9 0 0 0 23.3.3c18.8-.3 19-.3 18.8-2.5-.2-2 .3-2.2 7.5-2.8 36.8-3 44.2-4.4 61.2-11l20-7.8c27.4-10.6 56.7-27 82.6-46.3 13.4-10 35.6-31.2 45.7-43.7A251.5 251.5 0 0 0 665 375.3c.6-7.2 1.3-13.7 1.6-14.5s1.2-4.6 2-8.4a25 25 0 0 1 3.8-9.7c5-5.9 4.3-35.5-1.4-63.2-2.5-11.9-10.9-39-15.7-50.7a315 315 0 0 0-44.5-75.2c-9.2-11.6-37.9-40.5-42.4-42.7-1.6-.8-4.9-3-7.4-5.2a263.4 263.4 0 0 0-43.8-26.6 98 98 0 0 0-19.4-8.1c-1 0 .3 1.2 4.5 3.8 3.1 2 5.7 4 5.7 4.3 0 .9-.8 1.3-1.9 1.3-1 0-2.3-.3-3.6-1-1.2-.6-2.5-1-3.3-1a1 1 0 0 0-.5.1c-1 .7 5.6 7.7 10.1 10.4 1.7 1.2 3.2 2.5 3.2 3 0 .6-.4.9-1 .9-1.5 0-4.1-1.1-7.8-3.3a337.6 337.6 0 0 0-52.4-22.3A303.7 303.7 0 0 0 354.1 52c-15.8 0-32.3.8-42 2.2a202 202 0 0 1-19.3 2.5c-1.5 0-1.8-.3-1.8-.8 0-.6 1.3-1 3-1s4.7-.8 7-1.6 5.5-1.5 7-1.5 3.8-.8 5-1.8c2.1-1.7 4.1-8.4 3.2-11-.2-.7-.6-1-1.5-1-.7 0-1.7.2-3.2.5l-18 3.4c-15.5 3-32 7.5-44 12l-5.7 2.1 5.3 5.4c3 3 5.1 5.5 4.8 5.8-.2.2-3 1.1-6.4 2-8.8 2.5-22.5 8.7-22.5 10.3 0 .8.6 1.3 2 1.3M477 66c2.7 0 4-1.4 5.5-4.3 1-1.7 1.4-3 1.3-3l-8.8-3a319.1 319.1 0 0 0-59.9-16c-1 0-.4 6.5 1.1 9.2.5 1 2.1 2 3.6 2.3 9.8 2 38.4 9.3 47 12.2 5 1.7 8.1 2.6 10.3 2.6m-308.6 45.4c.8 0 7.6-6.3 15.5-14.2 8-8 14.3-14.5 14-14.5-2.8.3-16 10.5-25.8 20-5 4.7-6.3 7.8-3.7 8.7m-50.9 48.3c1.3 0 4.8-2.8 8.9-6.9 5-5.2 8.9-7.9 11-7.9.7 0 1.2.3 1.5.8.1.2.3.3.6.3.7 0 1.9-.7 3.5-2 1.9-1.8 3.1-3.5 2.6-3.9h-.3c-.7 0-2 .8-3.4 2.1-1.3 1.3-2.2 2-2.6 2-.3 0-.4-.3-.4-.6 0-2.8 4.1-8 8.7-11 6.2-4.2 18.3-16.8 18.3-19 0-.9-.7-2.2-1.6-3-.8-.7-2.2-2.5-3-4-.8-1.6-1.8-2.9-2.2-2.9-1.1.1-10.5 8.1-22.5 19.2a280.7 280.7 0 0 0-29.7 32.8c0 1 8 4 10.6 4m-71.8 151c4 0 5.3-2.4 7.2-12.8 1.9-10.7 6.1-25.8 11.1-40 1.6-4.6 2.9-8.8 2.9-9.2 0-.5-1.4-2.5-3.1-4.5-1.7-1.9-3.5-3.5-4-3.5-2.1 0-10 29.3-13.5 50-2.9 17.6-3 20-.6 20M343.6 0c-17.1 0-19 .8-20.5 1.4l-.1.1c-4.5 2-7.5 6-7.8 10.2 0 1 0 2 .3 3-2.6 2.8-3.4 7.5-3.7 11v1.6c-2 .4-4.6 1-7.6 1.4-18 3.2-40.6 8.8-53.6 13.5l-6.2 2.2L243 43a23 23 0 0 0-9.6-6.7 12.4 12.4 0 0 0-11.7-8.6c-1.8 0-3.2 0-15.2 6-6.7 3.4-14.4 7.5-20.5 11l-9.2 4.9c-3 1.5-10.6 6.4-16.4 10.3l-.2.2c-7.7 5.1-12 8-14 13-2.2 5.5-.8 10.6 1.4 18.5v.2c.7 2 1.2 4 1.6 5.4A439.6 439.6 0 0 0 103 143v.2a37.5 37.5 0 0 1-3 3.3c-3-.6-7.8-2-10.3-3a20 20 0 0 0-7.3-1.7c-7.7 0-11.5 5.8-19 17.3a145.4 145.4 0 0 0-9.5 14.8l-4.9 8.5c-3.4 6-7.8 14.8-10.9 21-7.1 14.7-7.1 16.1-7.1 18 0 3.4 1.6 5.8 8.2 12.3a128 128 0 0 0 8.4 7.7l-1.8 5.5c-5 15.9-9.7 36.7-12 54.4l-.6 3.7-1.3.4c-3.5 1-9.9 2.5-14.1 3.5l-1.5.4c-5.6 1.2-12 2.6-14.7 10.8C0 325 0 332.1 0 347.7v4.6c0 43.9 2.6 48.7 4 51.3v.3c1.8 3 5 5 12 7.2 4.7 1.6 10.5 3 16.4 4.2 4.3.8 6.9 1.4 8.4 1.8l2.4 8.8 13.1 41.9v.1l.9 2.2c-3 3.2-8.6 8.9-15.3 15.2C34.6 492 32 496 32 500.2c0 2 0 6.8 17 36 8 13.6 18.5 30.3 24.4 36.7 1.4 1.6 4.3 4.8 10.2 4.8 4.6 0 10.7-1.9 27-6.9a22 22 0 0 1 5-1.1 376.7 376.7 0 0 0 37 36.7l2 1.4-1 3c-1.3 4.3-3.4 10.8-4.7 14.3-2.2 6-5.8 16 1.9 23.3 8 7.3 40 26.2 56.6 34a45.7 45.7 0 0 0 18.3 5.8c4.8 0 8.9-2 12.4-5.9 1.6-1.7 5-5.5 4.8-10.2l4-.7 3.8-4c1.8-1.8 2.5-2.2 2.6-2.2h.2c1.5 0 4.6.8 7 1.6 30.1 9 48.9 13.4 64.7 15a536.2 536.2 0 0 0 82.3.2h1.4c2.6 1.5 4.2 1.7 5.3 1.7 3.3 0 6-2 7.4-4.7v-.2a360.1 360.1 0 0 0 54.4-14.4c1 .7 2 1.7 3.1 2.9 6.5 6.7 8.8 8.5 12.4 8.5 2.8 0 5.3-1.2 8.6-4.3 1-.7 3.2-2 4.6-2.6 3.3-1.1 7.7-3.6 10.3-5.4l.2-.1c1.6-1.1 3.5-2 4-2.3 2.3-.6 7.6-2.8 11.4-4.4l6.8-2.8 9.2-3.9h.2c8.2-3.5 26.8-14.6 30.6-20.5 2.4-3.7 3-8.7 1.6-13.4a14 14 0 0 0-4.8-7.4l4.4-4.3c5.3-5.3 12.6-12 16.4-15.4 8.2-7.1 12.9-12.6 14.4-21 .5-2.6 1.4-7.6-1.7-11.4l-.6-.6a325.6 325.6 0 0 0 80.5-178.1c2.5-20.7 2.4-56.2-.2-67.8v-.2c-.9-3.5-2.6-12.6-4-21.2a304.7 304.7 0 0 0-42.4-111c-15.4-25.2-35.2-48.1-66.1-76.9C559 86.2 518.5 61.7 493 51.4l-6.6-2.7-1.8-.8c-8-3.5-28.9-10.2-40.6-13-9.4-2.3-17.2-4-24.7-5.5.5-3-.5-6.6-1.6-10.5-2.3-9.3-3.7-11-5-12-1.6-1.5-3.5-2.4-14.8-4.5-9.2-1.5-38-2.4-54.3-2.4zm-85.5 59.3c7.7-2.5 16.7-5 25.8-7.2-.6 1.2-.9 2.5-.9 3.8 0 4.2 3 8.8 9.7 8.8 2.8 0 7.4-.6 20.5-2.6A322 322 0 0 1 354 60c11.6 0 22 .4 29 1.2 30.4 3.3 42.6 5.9 65.1 13.6a334.7 334.7 0 0 1 51 21.7c5.2 3 8.7 4.3 11.7 4.3a8.9 8.9 0 0 0 8.7-11.5c14.1 7.3 28.2 16 36.3 22.6 2.5 2 6.3 4.8 8.8 6 4.4 3 30.6 29 39.8 40.7a307.7 307.7 0 0 1 43.4 73.3c4.6 11.2 12.9 38 15.3 49.3 6 29 5 52.4 3 56.6a33 33 0 0 0-5.4 13l-1.7 7.3c-.4 1-1 3-2 16.5-3.8 45.6-23.1 93-53 129.9-9.7 12-31.2 32.6-44.3 42.3a352.6 352.6 0 0 1-80.7 45.2l-20 7.8c-16.1 6.3-22.7 7.5-59 10.5-3.8.4-6.7.6-9 1.5l-4.5-.3c-4-.2-9.8-.3-17.3-.3-44.9-.2-82.1-8.5-120.8-27a237.8 237.8 0 0 1-79.4-61 255.8 255.8 0 0 1-46.3-87.8 222 222 0 0 1-8.5-72.4c.6-22.8 2.5-35.6 8.3-57A229.8 229.8 0 0 1 216 178.1a281 281 0 0 1 35.4-19l-2.3 1.6a229.1 229.1 0 0 0-49.7 46.1c-8 10.3-11.1 15.7-8.4 21.1 1.4 3 4.5 4.9 8 4.9 3.2 0 5.6-1.3 12.5-8.9 19-20.2 29.5-30.4 38.8-37.6 7-5.5 18.4-13.4 21.2-15a276.2 276.2 0 0 1 46.2-22.5l.1-.1 7.3-3 12.4-5.3c16.4-7.7 16.7-7.8 33.1-12.4a249.7 249.7 0 0 1 74.1-5.3l7.2.3c6.7 0 18.5-.8 23.8-6 5.7-5.7 2.8-11.3 1.5-13.7l-.1-.2c-4.5-7.9-22.8-16.1-28.8-18.8a217.6 217.6 0 0 0-41-14.1c-15.1-3.2-35.6-5-57.7-5a305 305 0 0 0-57.2 4.8A284.7 284.7 0 0 0 252 81.2c.5-1.6.5-3.4 0-5 5.1-1.4 6.2-2 7.5-3.3a8.3 8.3 0 0 0 .9-10.6l-1.6-2.1-.8-.9z"/><path d="M516 136.8c-2.2 0-4.8-1.1-7.2-3.3-4.3-3.7-3.8-7-3.2-8.5 1.3-3.5 3.9-4 5.3-4 2.2 0 4.2 1.3 6.8 3.6 4.5 4 4.2 7.2 3.5 8.8a5.3 5.3 0 0 1-5.2 3.4z"/><path fill="#fff" d="M510.9 125c.9 0 2.2.8 4.2 2.6 3.4 3 3 5.2 1 5.2-1.2 0-2.8-.7-4.7-2.3-1.8-1.6-2.5-3-2.1-4 .4-1 .9-1.6 1.6-1.6m0-8c-1.6 0-6.7.5-9 6.8-.9 2-2 7.3 4.2 12.8 3.3 2.8 6.7 4.3 10 4.3 4 0 7.5-2.4 8.9-6 .8-2 1.9-7.4-4.6-13.2-2.7-2.3-5.6-4.7-9.5-4.7z"/><path d="M543.8 75.8c-1.2 0-1.6 0-22.8-10.3a551 551 0 0 0-23.5-10.9c-3.6-1.1-5.7-3-6.3-5.4-.5-2 .1-4 1.6-5.5.9-.8 2.4-1.9 5-1.9 3.7 0 9.7 2.2 21.2 7.8 12 5.8 17.4 9.2 22.2 13.7 6.2 5.6 7.1 7 6.5 9.5l-3 2.9-.9.1z"/><path fill="#fff" d="M497.8 45.8c3 0 9 2.3 19.5 7.4a80.7 80.7 0 0 1 21.2 13c3.3 3 5.7 5.5 5.3 5.6-.5 0-9.9-4.5-21-9.9-11.2-5.4-22-10.4-24-11-3.5-1.2-4.7-2.8-3.1-4.4.4-.4 1.1-.7 2.1-.7m0-8c-4 0-6.4 1.7-7.8 3.1a9.7 9.7 0 0 0-2.7 9.2c1.1 4.8 5.4 7.2 9 8.3 1.5.6 10.5 4.7 23 10.7 22 10.7 22.2 10.7 24.5 10.7l2-.2a8 8 0 0 0 5.8-5.9c1.3-5.2-2.3-8.5-7.7-13.4A88 88 0 0 0 521 46c-12.6-6-18.6-8.2-23.1-8.2z"/><path d="M717.7 379.3c-4 0-4.2-3.8-4.3-7.6a447.9 447.9 0 0 1 0-22.6c.1-3.6.2-7.6 4.3-7.6 2.4 0 3.9 1.6 4.3 4.8 1 7.5 1 22 0 28.6-.6 3.9-3 4.4-4.3 4.4z"/><path fill="#fff" d="m717.7 345.5.3 1.3a126.3 126.3 0 0 1-.3 28.5c-.3 0-.4-5-.4-15s.1-14.8.4-14.8m0-8a8 8 0 0 0-8 6.7l-.1 1.6-.2 3.2a442.6 442.6 0 0 0 .4 27.7c.8 4 4 6.7 8 6.7 1.1 0 7-.4 8.1-7.8a133.3 133.3 0 0 0-.3-31.5 8.1 8.1 0 0 0-7.9-6.6z"/><path d="M630.4 569.6c-1.4 0-2.7-.4-4.2-1.2-.6-.3-1.7-.6-2.1-.6-3.5 0-5.3-2-5.9-3.8-.5-1.7-.1-4.2 2.4-6 2.8-2.5 20.2-26.4 27.3-38 4.2-6.6 10.8-18.1 14.7-25.5 8.1-15.7 9.1-15.7 11.5-15.7 1.5 0 5 .7 5 7.5 0 6.4-17.3 38.1-25.6 52.5l-3.7 6.3-8 13.9c-3.3 5.8-6.5 10.6-11.4 10.6z"/><path fill="#fff" d="M674.1 482.8c.5 0 .9 1.6.9 3.5 0 4-11.9 28-25 50.5L638.3 557c-3.5 6.1-5.6 8.6-8 8.6-.7 0-1.4-.3-2.2-.7a10 10 0 0 0-4-1c-2.2 0-2.8-1.5-1.1-2.6a387 387 0 0 0 43.1-64.9 81 81 0 0 1 8-13.6m0-8c-5 0-7.1 3.7-9.6 8-1.6 2.6-3.6 6.2-5.5 9.9a464.5 464.5 0 0 1-14.5 25.1v.1A459.7 459.7 0 0 1 618 555a9.2 9.2 0 0 0-3.6 10.1c1.1 4 4.8 6.6 9.4 6.7l.5.2c2 1 4 1.5 6 1.5 7.7 0 11.9-7.2 15-12.5a3071 3071 0 0 1 11.6-20.2c6.4-11 12.8-22.9 17.7-32.7 8.4-17 8.4-20 8.4-21.8 0-1.1 0-2.8-.5-4.5-1.3-5.1-5.4-7-8.4-7z"/><path d="M642 572.8c-3.5 0-4.9-2-5.3-3-1.5-3-.3-6.5 4.2-12.5 2-2.7 5.8-8.8 8.3-13.4l2.4-4.3c2-3.8 4-7.4 5.2-9.3 1.5-2.8 3.2-6 3.8-7.4 1.3-3.1 3.6-5 6.1-5s5.3 2 5.3 6.5c0 4.7-10.3 23-14.3 29.3a100 100 0 0 0-2.9 4.8c-.3 1-1.2 2.4-3.7 6.6l-.3.4c-2.9 4.4-5.5 7.3-8.8 7.3z"/><path fill="#fff" d="M666.7 522c.7 0 1.3.7 1.3 2.4 0 2.5-7.3 17.1-13.7 27.2-1.8 3-3.3 5.4-3.3 5.7s-1.6 3-3.5 6c-2.4 3.6-4.2 5.5-5.5 5.5-3.1 0-2.4-3 2.1-9 2.2-3 6.1-9.3 8.6-14l7.5-13.5a107 107 0 0 0 4.1-7.8c.7-1.7 1.7-2.6 2.4-2.6m0-8c-4.2 0-7.8 2.8-9.8 7.5-.4 1-2 4-3.6 6.9l-5.2 9.3-2.4 4.3v.2c-2.4 4.4-6 10.2-8 12.8-4 5.4-7.4 11.1-4.6 16.7 1.6 3.3 5 5.2 9 5.2 5.6 0 9.1-4.7 12-9l.2-.3.2-.3a130 130 0 0 1 6.6-11.3c3.1-5 6.7-11.3 9.6-17.2 5.3-10.5 5.3-12.8 5.3-14.3 0-6.9-4.7-10.5-9.3-10.5z"/><path d="M653 575.7c-1.2 0-2.6-.3-4.1-1l-2.7-1.2-1-6.7 2.1-1.8a156 156 0 0 0 13.3-21c.5-1.3 2.4-5.2 6-5.2s7.4 3.8 7.4 7.6c0 1.1 0 2.5-7.5 14.7-7.4 12-8.8 13-9.5 13.4l-.2.2c-1.1.6-2.4 1-3.7 1z"/><path fill="#fff" d="M666.5 542.8c1.5 0 3.5 2.1 3.5 3.6a153 153 0 0 1-15.2 24.8c-.5.3-1.1.5-1.7.5-.8 0-1.6-.3-2.6-.7l-2.7-1.2 2.1-1.8c2.1-1.8 12.3-17.7 14.4-22.5.6-1.5 1.6-2.7 2.2-2.7m0-8c-4 0-7.6 2.8-9.6 7.6-1.7 3.9-10.3 17.2-12.4 19.7l-2 1.6 2 13.4 2.7 1.2c2.1 1 4 1.4 5.9 1.4 2 0 4-.5 5.7-1.6l.4-.2c1.3-.9 2.9-2 10.7-14.7 8.1-13.2 8.1-14.7 8.1-16.8 0-6-5.6-11.6-11.5-11.6z"/><path d="M507.5 688h-1.2c-.6-.2-2.3-.6-7.4-4.8-5.5-4.5-5.9-6-5.9-7.6 0-2.6 1.6-4 9-7.5 28.2-13.2 47.4-24 62.5-35.1 3.4-2.5 6-4 8.7-4 4.5 0 6.3 4.3 7 5.9 2.3 5.5 2.9 10.2-2 15.4-7 7.4-62 37.7-70.7 37.7z"/><path fill="#fff" d="M573.2 633c1.4 0 2.3 1.1 3.3 3.4 2.2 5.2 2 7.8-1.2 11.1-7.2 7.6-61.1 36.5-67.8 36.5h-.3a34.8 34.8 0 0 1-10.2-8.4c0-.4 3-2 6.8-3.9a375 375 0 0 0 63.1-35.5c3-2.1 4.8-3.2 6.3-3.2m0-8c-3.8 0-7 1.8-11 4.7a370 370 0 0 1-61.8 34.8c-6.4 3-11.4 5.4-11.4 11.1 0 4 2.4 6.6 7.3 10.7 6 4.8 8 5.3 9.2 5.5.6.2 1.3.3 2 .3 1.8 0 7 0 37.5-16.4 7-3.8 30.3-16.6 36.1-22.7 6.8-7.2 5.2-14 2.8-19.7-1.6-3.8-4.8-8.3-10.7-8.3z"/><path d="M368.1 692.5c-11.8 0-22.9-.9-31.3-2.3-4.8-.8-9.8-1.4-11.6-1.4-3.3 0-12.2 0-12.2-9.6v-3.5l6.4-3.2 2.8 2.1c3.3 2.4 11.1 3.3 36.4 3.8l14.7.2c26.7 0 34-2.2 35.5-2.9l3.5-1.4 5.5 4.3-.5 3.6a19 19 0 0 1-1.7 5.8l-.5.6c-1.1 1.2-2.4 1.6-7.7 1.8l-20.1 1.3c-6.1.5-12.6.8-19.2.8z"/><path fill="#fff" d="m317 675.7 2.8 2.1c4.2 3.1 11.4 4 38.7 4.6 5.2.2 10.2.2 14.8.2 18.8 0 32.2-1.1 37.1-3.2l3.4-1.4-.5 3.6c-.3 2-.8 4-1 4.2-.3.3-2.6.5-5.1.6l-20.2 1.3a234.5 234.5 0 0 1-49.5-1.5c-4.4-.7-9.9-1.4-12.3-1.4-6.2 0-8.2-1.4-8.2-5.6v-3.5m4.8-6.4-12.8 6.4v3.5c0 6.1 2.8 9.3 5.2 11 3.4 2.3 7.5 2.6 11 2.6 1.4 0 5.9.5 11 1.3a240.2 240.2 0 0 0 51.4 1.6l20-1.3c4.8-.2 8-.5 10.4-3l1-1.1c.3-.6 1.4-2.2 2.2-7.5l.5-3.7-11-8.5-3.4 1.4c-.7.3-6.8 2.6-34 2.6-4.6 0-9.5 0-14.6-.2-19.6-.4-31.4-1-34.1-3l-2.8-2z"/><path d="m363.4 716.4-5-.1c-10.8-.4-29-2-32.8-3.3-4.4-1.4-9.6-11.3-9.6-15.6 0-1.8 0-6.6 4.3-6.6l3.3.4 9.4 1.1a651 651 0 0 0 72.8 1.2h1.5c5.8 0 6.3 4.4 6.5 6.4.6 6.7-4.1 13-10.5 14.2-6.2 1.2-27.8 2.3-40 2.3z"/><path fill="#fff" d="m320.3 694.8 12.2 1.5a651.7 651.7 0 0 0 73.5 1.2h1.3c2 0 2.3.5 2.5 2.8.4 4.5-2.9 9.1-7.2 10a354 354 0 0 1-44.1 2c-11.1-.4-28.5-2-31.7-3-2.2-.8-6.8-8.7-6.8-11.9 0-1.4.1-2.6.3-2.6m0-8a8.1 8.1 0 0 0-7.8 6l-.4 2-.1 2.6c0 3.8 2 8 3.7 10.8 3.6 6.4 6.7 8 8.7 8.6a230.9 230.9 0 0 0 39 3.6c12.4 0 34-1.1 40.7-2.3a17.5 17.5 0 0 0 13.7-18.5v-.2c-.2-1.5-.5-4.1-2.4-6.4a9.9 9.9 0 0 0-8.1-3.5h-1.7c-5.4.2-12.8.3-21.6.3-21.4 0-44-.6-50.5-1.4l-9.3-1.2-3.9-.4z"/></svg>
```
Since we are not gonna rewrite it i think we can use optimizer on it. It's just slimmer by 30%